apiserver/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "apiserver.go",
+        "config.go",
+        "handlers.go",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//model:go_default_library",
+        "@com_github_emicklei_go_restful//:go_default_library",
+        "@com_github_gogo_protobuf//proto:go_default_library",
```

Comment:
Please use golang protobuf, not gogo protobuf. Auto-import tools tend to choose gogo for some reason.